src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -843,16 +842,14 @@ public boolean canBeInstantiated() {
 					protected OfflinePlayer deserialize(final Fields fields) throws StreamCorruptedException {
 						if (fields.contains("uuid") && uuidSupported) {
 							final UUID uuid = fields.getObject("uuid", UUID.class);
-							OfflinePlayer p;
-							if (uuid == null || (p = Bukkit.getOfflinePlayer(uuid)) == null)
+							if (uuid == null)
 								throw new StreamCorruptedException();
-							return p;
+							return Bukkit.getOfflinePlayer(uuid);
```

Comment:
has getOfflinePlayer always been guaranteed to be non-null? if not, this may not be the same as old behavior

Reply:
Bukkit.getOfflinePlayer(uuid) is @NotNull

Reply:
Same thing here,
![image](https://user-images.githubusercontent.com/19924575/88004850-7b931580-cb3a-11ea-9126-413cbfe95991.png)